runtime/compiler/runtime/RelocationRuntime.cpp

```diff
@@ -1057,7 +1057,7 @@ TR_SharedCacheRelocationRuntime::checkAOTHeaderFlags(const TR_AOTHeader *hdrInCa
    {
    bool defaultMessage = true;
 
-   if (!TR::Compiler->target.cpu.isCompatible(hdrInCache->processorDescription))
+   if (!TR::Compiler->relocatableTarget.cpu.isCompatible(hdrInCache->processorDescription))
```

Comment:
Strictly speaking, the `target` should be a superset of the `relocatableTarget`, so even if the relocatable target of the code in the SCC wasn't compatible with the relocatable target of the running JVM, but it **is** compatible with the target of the running JVM, it is ok to run the code. However, we probably don't want to allow such a scenario, so I figure it's probably best to use the relocatable target.